FocusOn/View/ProgressView.swift

```diff
@@ -6,15 +6,42 @@
 //
 
 import SwiftUI
+import SwiftUICharts
 
 struct ProgressView: View {
+    @StateObject var viewModel = ProgressViewModel()
+    
+    @State private var chartData: [DataPoint]?
+    
     var body: some View {
-        Text("Progress View")
+        VStack {
+            Text("Here is your progres")
+            HStack(spacing: 0) {
+                BarChartView(dataPoints: chartData ?? [])
+                    .chartStyle(
+                        BarChartStyle(
+                            barMinHeight: 10,
+                            showAxis: false,
+                            showLegends: true
+                        )
+                    )
+            }
+            .padding(20)
```

Comment:
You can have implement a pagination style thanks to to the style `page` of the component `TabView`, [here](https://www.hackingwithswift.com/quick-start/swiftui/how-to-create-scrolling-pages-of-content-using-tabviewstyle) is a nice tutorial that should help.

If it doesn't, here is a piece of code that works:
```
Text("Here is your progres")
    .foregroundColor(.accentColor)
    .font(.largeTitle)

let chunkedChartData = chartData.chunked(into: 7)
let numberOfWeek = chunkedChartData.count
TabView {
    ForEach(0..<numberOfWeek, id: \.self) { weekNumber in
        ProgressPageView(
            title: "Week \(weekNumber)",
            dataPoint: chunkedChartData[weekNumber]
        )
    }
}
.tabViewStyle(.page)

private struct ProgressPageView: View {

    let title: String
    let dataPoint: [DataPoint]

    var body: some View {
        VStack(
            alignment: .leading,
            spacing: 20
        ) {
            Text(self.title)
                .foregroundColor(.accentColor)
                .font(.largeTitle)
            HStack {
                BarChartView(dataPoints: self.dataPoint)
                    .chartStyle(
                        BarChartStyle(
                            barMinHeight: 10,
                            showAxis: false,
                            showLegends: false
                        )
                    )
                    .frame(maxWidth: .infinity, maxHeight: .infinity)
            }
            .padding(20)
        }
    }
}
```

Note: in the above code, I created a custom view in order to reduce the code size of `ProgressView.body`, this is not mandatory, but I advice you to do the same and to try to take the habit of reducing code size.